front/app/components/admin/GraphCards/ActiveUsersCard/useActiveUsers/typings.ts

```diff
@@ -7,34 +7,6 @@ import {
 
 export type QueryParameters = ProjectId & Dates & Resolution;
 
-// Response
```

Comment:
Now this uses the `graph_data_units` hook, so we can use the types from that folder instead